src/Hl7.Fhir.Specification/Specification/Snapshot/ElementDefinitionExtensions.cs

```diff
@@ -64,16 +64,15 @@ public static IEnumerable<string> PrimaryTypeProfiles(this ElementDefinition ele
             return elem.Type.FirstOrDefault()?.Profile ?? Enumerable.Empty<string>();
         }
 
-        /// <summary>Returns the type code of the primary element type, or <c>null</c>.</summary>
-        public static FHIRAllTypes? PrimaryTypeCode(this ElementDefinition elem)
+        /// <summary>Returns the typeName of the primary element type, or <c>null</c>.</summary>
+        public static string? GetTypeCode(this ElementDefinition elem)
```

Comment:
Ah, one more change to the public API. Have you been tracking them on a gist or wiki page?

Reply:
Yes, a list of breaking changes is necessary.